cellpose/gui/guiparts.py

```diff
@@ -451,6 +451,16 @@ def __init__(self, parent=None):
         # This marks the menu button as checked
         parent.minimapWindow.setChecked(True)
 
+        # Create and add a highlight rectangle to the minimap with initial position [0, 0] and size [100, 100], outlined
+        # in white with a 3-pixel width.
+        self.highlight_area = pg.RectROI([0, 0], [100, 100], pen=pg.mkPen('w', width=3), resizable=False, movable=False)

```

Comment:
I would be good to change this size to the size of the minimap if possible (see picture in the main review)